pkg/insight/insightstore/deploymentstore.go

```diff
@@ -26,14 +26,14 @@ var (
 )
 
 type DeploymentStore interface {
-	// GetDeployments returns slice of Deployment sorted by startedAt ASC.
-	GetDeployments(ctx context.Context, projectID string, from, to int64, minimumVersion *model.InsightDeploymentVersion) ([]*model.InsightDeployment, error)
+	// List returns slice of Deployment sorted by startedAt ASC.
+	List(ctx context.Context, projectID string, from, to int64, minimumVersion *model.InsightDeploymentVersion) ([]*model.InsightDeployment, error)
 
 	PutDeployments(ctx context.Context, projectID string, deployments []*model.InsightDeployment, version *model.InsightDeploymentVersion) error
```

Comment:
imo, we can also fix it like this
```suggestion
	Put(ctx context.Context, projectID string, deployments []*model.InsightDeployment, version *model.InsightDeploymentVersion) error
```